test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -404,6 +404,24 @@ public async Task CheckForNonTSqlLanguageTest(string scriptText, bool expectedRe
             var result = await langService.CheckForNonTSqlLanguage(scriptFile.ClientUri, parseResult);
 
             Assert.AreEqual(expectedResult, result, message);
+            return result;
+        }
+
+        [Test]
+        [TestCase(
+            "./TestScript.txt", 
+            false, 
+            "Long Sql Server script; should not detect Non-T-SQL syntax"
+        )]
+        public async Task CheckForNonTSqlLanguageFromFilePathTest(string scriptText, bool expectedResult, string message)
+        {            
+            if (File.Exists(scriptText))
```

Comment:
Test should fail if the file doesn't exist.  If someone moves files around so that it's no longer in the right spot, then this test will pass.